db/seeds/hanna_barbera.rb

```diff
@@ -0,0 +1,44 @@
+# Creates a number of users themed from Hanna-Barbera cartoon characters.
+
+PASSWORD = 'password'
+filepath = File.join(Rails.root, 'db', 'seeds', 'hanna_barbera.yml')
+
+
+yaml_root = YAML.load_file(filepath)
+master_group = Group.find_or_create_by_name('hanna_barbera_group')
+
+yaml_root['users'].each do |username, values|
+  user_info = values.slice('first_name', 'last_name', 'full_name',
+                           'title', 'suffix')
+  # Create users...
+  user = User.find_or_create_by_username(username)
```

Comment:
I suggested `FactoryGirl.create` because in case the model changes and requires more fields, this script won't fail.
I guess we won't accidentally run this script on non-development environments given the script name, so I guess it's ok?
Usually I just wipe the db and populate it again in development.